DependencyInjection/CompilerPass/RegisterCompilerPass.php

```diff
@@ -115,8 +113,21 @@ private function registerClass(ContainerBuilder $container, $prefix, $className)
         $definition = $container->register($id, $className);
         $definition->setAutowired(true);
 
-        if ('command' === $prefix) {
+        $this->customizeServiceDefinition($definition);
+    }
+
+    private function customizeServiceDefinition(Definition $definition)
+    {
+        $class = $definition->getClass();
+
+        // Commands
+        if (is_subclass_of($class, Command::class)) {
             $definition->addTag('console.command');
         }
+
+        // Event subscribers
+        if (is_subclass_of($class, EventSubscriberInterface::class)) {
+            $definition->addTag('kernel.event_subscriber');
+        }
```

Comment:
done.
``Configuration`` is a bit more complicated but it allows us to support 3 syntaxes:
``console.command``, ``[console.command]`` or ``[[console.command, []], 'kernel.event_subscriber]``